edit-post/components/plugins-panel/style.scss

```diff
@@ -12,6 +12,10 @@
 
     h3 {
       margin: 0;
+      font-weight: 400;
+      color: inherit;
```

Comment:
Haven't had time to check, but should the font size also be addressed? The font size should be 13px, and you get that from the $default-font-size variable here https://github.com/WordPress/gutenberg/blob/master/edit-post/assets/stylesheets/_variables.scss#L7

Reply:
<img width="920" alt="schermafbeelding 2018-02-21 om 09 57 55" src="https://user-images.githubusercontent.com/17744553/36471160-bf7ecb76-16ed-11e8-8d1f-a265306b43a1.png">
@jasmussen The current font size is 13px. Or do I need to set it explicitly for some reason?

Reply:
Ah right, sorry, wasn't able to test. No that looks good. Though feel free to set the font weight to 600 so it's bold, like an active tab. Otherwise 👍 👍